provisioning/miniprov/hare_mp/main.py

```diff
@@ -395,13 +407,14 @@ class ProcessStartInfo:
     fid: str
 
 
+@func_log(func_enter, func_leave)
 def start_mkfs(proc_to_start: ProcessStartInfo) -> int:
     try:
-        logging.debug('Starting mkfs process [fid=%s] at hostname=%s',
-                      proc_to_start.fid, proc_to_start.hostname)
+        logging.info('Starting mkfs process [fid=%s] at hostname=%s',
+                     proc_to_start.fid, proc_to_start.hostname)
         command = proc_to_start.cmd
         execute(command)
-        logging.debug('Started mkfs process [fid=%s]', proc_to_start.fid)
+        logging.info('Started mkfs process [fid=%s]', proc_to_start.fid)
         rc = 0
     except Exception as error:
         logging.error('Error launching mkfs [fid=%s] at hostname=%s: %s',
```

Comment:
While testing also please check and confirm if the .error, .warning etc messages are getting logged properly.
For checking it, I think either the erroneous conditions can be simulated or simply .error, .warning etc logs can be added (temporarily for testing) in the happy path.
This is just to ensure that, if some error / warning occurs, it should not error out for logging related issue or fail to log the important error/warning message.
Thanks!

Reply:
![image](https://user-images.githubusercontent.com/62925646/149182685-381ea62a-cfe5-446a-be80-af213597cb5f.png)
I added the above change in happy path to check the working of logging.error() and logging.warn().
To test the changes, I used post_install phase of hare_setup.
Following is the screenshot from the log.

![image](https://user-images.githubusercontent.com/62925646/149183166-9e9b002c-2554-4c5d-8550-8ec1e4e03478.png)